core/src/main/java/org/hibernate/ogm/persister/impl/OgmEntityPersister.java

```diff
@@ -190,10 +190,12 @@
 			throw log.getIdentityGenerationStrategyNotSupportedException( getEntityName() );
 		}
 
-		tableName = persistentClass.getTable().getQualifiedName(
-				factory.getDialect(),
-				factory.getSettings().getDefaultCatalogName(),
-				factory.getSettings().getDefaultSchemaName()
+		tableName = gridDialect.getNamingAdaptor().makeValidTableName(
```

Comment:
Some more business around table names is going on further below in that constructor. Should that contract be applied to those as well? I'm not sure, as this stuff might be "dead" and not used for OGM atm.